README.md

```diff
@@ -1,2 +1,9 @@
 # quilter
+
 A handy utility for downloading and merging datasets.
+
+## Usage (WIP)
```

Comment:
This will need development instructions as well as installation instructions

Reply:
Coming in next commit.